docs/source/data/data_catalog.md

```diff
@@ -720,125 +135,41 @@ cars:
-This use is not recommended unless you are prototyping in notebooks.
-```
-
-#### Save data to memory
-
-```python
-from kedro.io import MemoryDataSet
-
-memory = MemoryDataSet(data=None)
-io.add("cars_cache", memory)
-io.save("cars_cache", "Memory can store anything.")
-io.load("cars_cache")
+```{note}
+Note that HTTP(S) is a supported file system in the dataset implementations, but if you it, you can't also use versioning.
 ```
 
-#### Save data to a SQL database for querying
-
-We might now want to put the data in a SQLite database to run queries on it. Let's use that to rank scooters by their mpg.
-
-```python
-import os
-
-# This cleans up the database in case it exists at this point
-try:
-    os.remove("kedro.db")
-except FileNotFoundError:
-    pass
-
-io.save("cars_table", cars)
-ranked = io.load("scooters_query")[["brand", "mpg"]]
-```
+## Use the Data Catalog within Kedro configuration
 
-#### Save data in Parquet
+Kedro configuration enables you to organise your project for different stages of your data pipeline. For example, you might need different Data Catalog settings for development, testing, and production environments.
 
-Finally, we can save the processed data in Parquet format.
+By default, Kedro has a `base` and a `local` folder for configuration. The Data Catalog configuration is loaded using a configuration loader class which recursively scans for configuration files inside the `conf` folder, firstly in `conf/base` and then in `conf/local` (which is the designated overriding environment). Kedro merges the configuration information and returns a configuration dictionary according to rules set out in the [configuration documentation](../configuration/configuration_basics.md).
 
-```python
-io.save("ranked", ranked)
-```
+In summary, if you need to configure your datasets for different environments, you can create both `conf/base/catalog.yml` and `conf/local/catalog.yml`. For instance, you can use the `catalog.yml` file in `conf/base/` to register the locations of datasets that would run in production, while adding a second version of `catalog.yml` in `conf/local/` to register the locations of sample datasets while you are using them for prototyping data pipeline(s).
 
-```{warning}
-Saving `None` to a dataset is not allowed!
+To illustrate this, consider the following catalog entry for a dataset named `cars` in `conf/base/catalog.yml`, which points to a csv file stored in a bucket on AWS S3:
+```yaml
+cars:
+  filepath: s3://my_bucket/cars.csv
+  type: pandas.CSVDataSet
+ ```
+You can overwrite this catalog entry in `conf/local/catalog.yml` to point to a locally stored file instead:
+```yaml
+cars:
+  filepath: data/01_raw/cars.csv
+  type: pandas.CSVDataSet
 ```
+In your pipeline code, when the `cars` dataset is used, it will use the overwritten catalog entry from `conf/local/catalog.yml` which points to the local file.` dataset and rely on Kedro to detect which definition of `cars` dataset to use in your pipeline.
```

Comment:
```suggestion
In your pipeline code, when the `cars` dataset is used, it will use the overwritten catalog entry from `conf/local/catalog.yml` and rely on Kedro to detect which definition of `cars` dataset to use in your pipeline.
```

Reply:
Thanks @ankatiyar ! Slight change in the final paragraph, please could you take a look as there was a typo there, hopefully I've fixed it correctly?

Reply:
Yep, looks goo! Thanks for fixing it.